### ISSUE TITLE
Failing builds

### ISSUE THREAD

**Issue by MSF-Jarvis** [jekyll/jekyll#4820]
https://gist.github.com/MSF-Jarvis/5d7fc877705054b2f115ab1a693226e0

###### What version of Jekyll are you using (`jekyll -v`)?
Default installed by Travis-CI

###### What operating system are you using?
Trusty Tahr

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, and the command you ran)
Log linked above

###### What did you expect to see?

Successful build

###### What did you see instead?

Failed build

**Comment by envygeeks** [comment]
Please update your Jekyll, and remove any ERB, we do not support Liquid (in front-matter) or ERB (actually we don't support ERB at all.)   We no longer support Jekyll 2.4 and there will be no more releases so please update it to the latest Jekyll (or if you use pages use the github-pages gem.)

**Comment by parkr** [comment]
@MSF-Jarvis Check out [our Continuous Integration guide](https://jekyllrb.com/docs/continuous-integration/). There is a piece that notes this: "Be sure to exclude vendor from your  _config.yml".
TL;DR: add `exclude: [vendor]` to your `_config.yml`. Thanks!
